content/intro-to-storybook/ember/en/get-started.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Storybook for Ember tutorial'
+tocTitle: 'Get started'
+description: 'Setup Storybook in your development environment'
+---
+
+Storybook runs alongside your app in development mode. It helps you build UI components isolated from the business logic and context of your app. This edition of Learn Storybook is for Ember; other editions exist for [React](/react/en/get-started), [React Native](/react-native/en/get-started), [Vue](/vue/en/get-started), [Angular](/angular/en/get-started) and [Svelte](/svelte/en/get-started).
+
+![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
+
+## Setup Ember Storybook
+
+We’ll need to follow a few steps to get the build process set up in your environment. To start with, we want to use [ember-cli](https://github.com/ember-cli/ember-cli) to setup our build system, and enable [Storybook](https://storybook.js.org/) and [Qunit](https://qunitjs.com/) testing in our app. Let’s run the following commands:
+
+```bash
+# Create our application:
+ember new taskbox
+
+cd taskbox
+
+# Add Storybook:
+npx -p @storybook/cli sb init
+
+# Add Ember storybook adapter
+ember install @storybook/ember-cli-storybook
+```
+
+<div class="aside">
+<p>
+At the time of writing this tutorial adding the @storybook/ember-cli-storybook, will yeld the following message :
```

Comment:
```suggestion
At the time of writing this tutorial adding the @storybook/ember-cli-storybook, will yield the following message :
```

Reply:
It was fixed with the latest commit. Going to mark it as resolved if you don't mind.